pkg/webhooks/workload_webhook.go

```diff
@@ -182,6 +184,87 @@ func validateContainer(c *corev1.Container, path *field.Path) field.ErrorList {
 	return allErrs
 }
 
+func validateAdmissionChecks(obj *kueue.Workload, basePath *field.Path) field.ErrorList {
+	var allErrs field.ErrorList
+	for i := range obj.Status.AdmissionChecks {
+		admissionChecksPath := basePath.Index(i)
+		ac := &obj.Status.AdmissionChecks[i]
+		allErrs = append(allErrs, validatePodSetUpdates(ac, obj, admissionChecksPath.Child("podSetUpdates"))...)
+	}
+	return allErrs
+}
+
+func validatePodSetUpdates(acs *kueue.AdmissionCheckState, obj *kueue.Workload, basePath *field.Path) field.ErrorList {
+	var allErrs field.ErrorList
+
+	knowPodSets := make(sets.Set[string], len(obj.Spec.PodSets))
+	for i := range obj.Spec.PodSets {
+		name := obj.Spec.PodSets[i].Name
+		knowPodSets.Insert(name)
+	}
+	for i := range acs.PodSetUpdates {
+		psu := &acs.PodSetUpdates[i]
+		psuPath := basePath.Index(i)
+		for _, msg := range validation.IsDNS1123Label(psu.Name) {
+			allErrs = append(allErrs, field.Invalid(psuPath.Child("name"), psu.Name, msg))
+		}
```

Comment:
We don't really care if it's `IsDNS1123Label` as long as the name is found in the list of podsets.

Reply:
removed https://github.com/kubernetes-sigs/kueue/pull/1220/commits/0943f326b3dc45a32e58f6899a8bbf4b755e07db